lib/hd/hd.js

```diff
@@ -46,18 +46,17 @@ HD.generate = function generate() {
 
 /**
  * Generate an {@link HDPrivateKey} from a seed.
- * @param {Object|Mnemonic|Buffer} options - seed,
- * mnemonic, mnemonic options.
+ * @param {Buffer} seed - seed,
  * @returns {HDPrivateKey}
  */
 
-HD.fromSeed = function fromSeed(options) {
-  return HDPrivateKey.fromSeed(options);
+HD.fromSeed = function fromSeed(seed) {
+  return HDPrivateKey.fromSeed(seed);
 };
 
 /**
  * Instantiate an hd private key from a mnemonic.
- * @param {Mnemonic|Object} mnemonic
+ * @param {Mnemonic} mnemonic
```

Comment:
here too, sorry - `fromMnemonic()` doesn't parse an options object any more it only parses a `Mnemonic`